src/handling/world/CharacterTransfer.java

```diff
@@ -32,7 +32,7 @@
 
     public int characterid, accountid, fame, pvpExp, pvpPoints, hair, face,
             faceMarking, elf, mapid, honourexp, honourlevel, guildid,
-            partyid, messengerid, ACash, nxCredit, MaplePoints,
+            partyid, messengerid, ACash, nxCredit, MaplePoints, RewardPoints,
```

Comment:
Although MaplePoints is capitalized, I think its confusing to use CamelCase'd variable names.